zadanie_3.js

```diff
@@ -0,0 +1,13 @@
+const average = (...args) => {
+    let sum = 0;
+    const arrayLength = args.length;
+
+    args.forEach((value) =>
+        sum += value
+    );
+    console.log(`The average value = ${sum/arrayLength}`);
+};
+
+average(4, 6, 5);
+average(1, 3, 6, 6);
+average(3, 8);
```

Comment:
możesz spróbować użyć https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/Reduce ale to jako zadanie dodatkowe :)